### DISCUSSION TITLE
Switch between monitors?

### DISCUSSION THREAD

**Discussion by naddel91** [LizardByte/community#427] [Sunshine]
Since I do not have a keyboard on the remote machine, how do I switch/toggle between monitors?

Best wishes!

**Comment by XPRAMT** [comment]
[https://github.com/Axixi2233/moonlight-android](https://github.com/Axixi2233/moonlight-android)

**Comment by willow-sprite** [comment]
if you mean virtual desktops, then you can install hacker's keyboard from fdroid for android.

this program emulates a full us qwerty keyboard with function and modifier keys (ctrl, tab, esc, arrow keys, windows button, etc.)

if you mean actually changing the monitor being displayed, which I can't imagine a use for, then you could change the monitor in sunshine settings then close sunshine, and use a terminal with a sleep timer to open sunshine with your changes applied.
